package.json

```diff
@@ -37,8 +37,8 @@
   },
   "homepage": "https://github.com/asciidoctor/asciidoctor-cli.js",
   "dependencies": {
+    "asciidoctor": "^1.5.6-rc.1",
```

Comment:
@mojavelinux I think I merged too quickly, the dependencies should be `asciidoctor.js`. I'm guessing that you have a local dependency named `asciidoctor` ?

Reply:
Oops. My mistake. Nope, that was just a typo.